graspologic/models/edge_swaps.py

```diff
@@ -0,0 +1,86 @@
+from typing import Union
+
+import numba as nb
+import numpy as np
+from scipy.sparse import lil_matrix
+
+from graspologic.types import Tuple
+
```

Comment:
should reference the original code this is based on

Reply:
explicitly say somewhere in the code comments where you based some of this on

Reply:
can you clarify what you mean by this?